js/src/get-started-page/features-card/index.js

```diff
@@ -74,27 +123,31 @@ const FeaturesCard = () => {
 						variant="body"
 					>
 						{ __(
-							`Showcase eligible products to shoppers looking for what you offer and drive traffic to your store with Google's free listings.`,
+							'Connect your Google Ads account, choose a budget, and Google will optimize your ads so they appear at the right time and place. ',
 							'google-listings-and-ads'
 						) }
 					</Text>
+					<LearnMoreLink
+						linkId="get-started-features-google-ads-learn-more"
+						href="https://woocommerce.com/document/google-listings-and-ads/#google-smart-shopping-campaigns"
```

Comment:
❔ 
Similar question to the above. A more relevant link to the content seems https://woocommerce.com/document/google-listings-and-ads/#google-performance-max-campaigns. Is the current one the correct link?

Reply:
@j111q  or @dominiquevias could you please check if we need to update the links? Thanks!

Reply:
Thank you for catching and for the link suggestions. I reviewed the copy against the links and they work. I updated the [marketing copy document](https://docs.google.com/document/d/1TUeMIQakEGzR8Qb45Y-kpVcqIzhKF0bAwb6bccMLWoA/edit?resourcekey=0-06jarg05meLrSMQC1M9N-w#) with the suggested links.

Reply:
Thanks @dominiquevias and @eason9487! Updated in d8c16ba.